bokeh/core/state.py

```diff
@@ -39,6 +39,8 @@ class State(object):
 
     '''
 
+    valid_notebook_types = ['jupyter', 'zeppelin']
```

Comment:
suggest module level tuple, and all caps `NOTEBOOK_TYPES = ('jupyter', 'zeppelin')`